lib/api_client.ex

```diff
@@ -32,6 +32,10 @@ defmodule GlassFactoryApi.ApiClient do
   end
 
   defp url(resource) do
-    "https://#{@access_info[:subdomain]}.glassfactory.io/api/public/v1/#{resource}"
+    "#{api_domain()}/api/public/v1/#{resource}"
+  end
+
+  defp api_domain do
```

Comment:
You could use a module attribute here `@api_domain` - it wouldn't make a big difference, it's just an idea 😅

Reply:
I've made with a method because of [this part of the tests](https://github.com/Cendrao/glass_factory_api/pull/3/files#diff-61dbe29746e78b5a7983dd6c37e90c9fR9-R13), if I use a module attribute it will be defined in compilation time and will not change in my test setup.